src/ui/src/core/index.ts

```diff
@@ -69,6 +75,15 @@ export function generateCore() {
 	 * @returns
 	 */
 	async function initSession() {
+		const localStorageItems = {};
+		for (let i = 0; i < localStorage.length; i++) {
+			const key = localStorage.key(i);
+			if (key.startsWith("wf.")) {
+				const value = localStorage.getItem(key);
+				localStorageItems[key.replace("wf.", "")] = JSON.parse(value);
+			}
+		}
```

Comment:
If one value is not valid JSON, it'll throw and stop the execution of the `initSession`. It might be better to catch it and remove the entry if so.

It could be something like

```js
Object.entries(localStorage)
  .filter(([key]) => key.startsWith("wf."))
  .reduce((acc, [key, value]) => {
    try {
      acc[key.replace('wf.', '')] = JSON.parse(value)
    } catch {
      localStorage.removeItem(key)
    }
  }, {});
```